tests/portability/test_pyinstaller_binary.py

```diff
@@ -42,11 +42,16 @@ def test_algokit_init(
     cli_path: str,
     dummy_algokit_template_with_python_task: dict[str, Path],
 ) -> None:
+    # TODO: revisit to improve
```

Comment:
To answer the question of adding to the PATH, you can do this in the GitHub action.
```
echo "$TEST_CLI_PATH" >> $GITHUB_PATH
```

Reply:
To be addressed separately, explicit notes are added in the codebase